app/controllers/application_controller.rb

```diff
@@ -14,7 +14,7 @@ class ApplicationController < ActionController::Base
   # Set auto papertrail
   before_action :set_paper_trail_whodunnit
 
-  rescue_from RuntimeError, Exception, with: :render_500 unless Rails.env.development?
+  # rescue_from RuntimeError, Exception, with: :render_500 unless Rails.env.development?
```

Comment:
Although I do disagree with catch-all rescuing, why just comment this out?

Reply:
We ("we" being Tom and I, since we did a lot of the recent commits together) were using this via d8bdc10 to get travis to throw errors instead of just returning the 500 page (so we could see what the errors were)

Reply:
It's redundant anyway seeing as we have exception notifier (i think) and rails (or apache or passenger i don't know which) will automatically load the 500 page anyway